lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1374,10 +1374,11 @@ func (w *WalletKit) ListSweeps(ctx context.Context,
 	// can match our list of sweeps against the list of transactions that
 	// the wallet is still tracking. Sweeps are currently always swept to
 	// the default wallet account.
-	transactions, err := w.cfg.Wallet.ListTransactionDetails(
-		in.StartHeight, btcwallet.UnconfirmedHeight,
-		lnwallet.DefaultAccountName, 0, -1,
-	)
+	transactions, firstIdx, lastIdx, err :=
```

Comment:
nit: could shorten `transactions` to just `txns` to avoid needing to format the call like this (which is correct, I just try to avoid it whenever possible as it looks off at first glance IMO).